modules/tinymce/changelog.txt

```diff
@@ -3,6 +3,7 @@ Version 5.4.2 (TBD)
     Fixed a regression where the `anchor_top` and `anchor_bottom` settings weren't working when set to `false` #TINY-6256
     Fixed an exception thrown when positioning the context toolbar on Internet Explorer 11 in some edge cases #TINY-6271
     Fixed list toolbar buttons not showing as active when a list is selected #TINY-6286
+    Fixed an issue where dialog boxes would sometimes not be shown or hidden when calling show or hide methods on the editor #TINY-6048
```

Comment:
This isn't specific to the dialogs, but the editor as a whole. So it should probably just be something like:
> Fixed an issue where the UI would sometimes not be shown or hidden when calling the show or hide methods on the editor #TINY-6048